app/models/person.rb

```diff
@@ -103,6 +103,8 @@ class Person < ActiveRecord::Base
   has_many :authored_comments, :class_name => "Comment", :foreign_key => "author_id", :dependent => :destroy
   has_many :community_memberships, :dependent => :destroy
   has_many :communities, -> { where("community_memberships.status = 'accepted'") }, :through => :community_memberships
+  has_one  :community_membership, :dependent => :destroy
+  has_one  :community, -> { where("community_memberships.status= 'accepted'") }, through: :community_membership
```

Comment:
Sounds like a good plan. `accepted_community` describes the meaning well. It might be that we need to also add plain `community`, but we can add that later if we need to.